spec/javascripts/angular/controllers/single_widget_ctrl_spec.js

```diff
@@ -0,0 +1,18 @@
+var servers = [];
+var graphBlob = {};
+describe('SingleWidgetCtrl', function() {
+  var $controller;
+  var scope;
+  beforeEach(inject(function(_$controller_, $rootScope){
+    $controller = _$controller_;
+    scope =  $rootScope;
+  }));
+
+  it('loads correctly', function() {
+    var $scope = scope.$new(true);
+    var controller = $controller('SingleWidgetCtrl', { $scope: $scope });
+    expect($scope.servers).toEqual([]);
+    expect($scope.globalConfig.aspectRatio).not.toBe(undefined);
```

Comment:
Did these tests catch the issue? Bonus points for red-green testing (http://en.wikipedia.org/wiki/Test-driven_development#Development_style).